projects/mac_ai/visualizations/llm_load_test/store/lts_parser.py

```
@@ -85,8 +85,7 @@ def generate_lts_settings(lts_metadata, results, import_settings):
     lts_settings.version = results.test_config.get(version_config_key) \
         if version_config_key else "Unknown"
 
-    if "ramalama" in lts_settings.platform and not lts_settings.version and results.ramalama_commit_info:
-        git_ref = results.test_config.get("prepare.ramalama.repo.git_ref")
+    if "ramalama" in lts_settings.platform and results.ramalama_commit_info:
         lts_settings.version = f"{git_ref}-{results.ramalama_commit_info.date_id}"
```

Flake8 (7.2.0)

[error] undefined name 'git_ref'

(F821)